docs/extend/plugins_graphdriver.md

```diff
@@ -34,8 +58,18 @@ Plugins are not required to accept these options as the Docker Engine does not
 require that the plugin use this path or options, they are only being passed
```

Comment:
All of this passive voice makes this hard to understand. Maybe something like "The array of option is passed through to the plugin, but the plugin is not required to parse or honor them."